app-emulation/qt-virt-manager/qt-virt-manager-9999.ebuild

```diff
@@ -12,7 +12,7 @@ if [[ ${PV} == "9999" ]] ; then
 	SRC_URI=""
 	KEYWORDS=""
 else
-	SRC_URI="https://github.com/F1ash/qt-virt-manager/archive/${P}.tar.gz"
+	SRC_URI="https://github.com/F1ash/qt-virt-manager/archive/${PV}.tar.gz"
```

Comment:
If that download uri is correct, you will have to rename the file to `${P}.tar.gz` in order to be valid.